server/src/main/java/com/epam/aidial/core/server/controller/ControllerSelector.java

```diff
@@ -75,6 +74,8 @@ public class ControllerSelector {
 
     private static final Pattern USER_INFO = Pattern.compile("^/v1/user/info$");
 
+    private static final Pattern APP_SCHEMAS = Pattern.compile("^/v1/ops/custom_application_schemas(/list|/schema)?$");
```

Comment:
Needs renaming. Something like this. Or whatever that makes it more clear
1) get_schema - gets a schema by schema id
2) list_schemas - lists all schemas
3) get_schema_descriptor - describes all schemas